djedi/plugins/form.py

```diff
@@ -45,13 +45,14 @@ class FormsBasePlugin(DjediPlugin):
     def forms(self):
         return {}
 
-    def get_editor_context(self, **context):
-        context.update(
-            {"forms": {
-                tab: form()
-                for tab, form in self.forms.items()
-            }}
-        )
+    def get_editor_context(self, request, **context):
+        if not request.is_ajax():
```

Comment:
what happens if request is ajax? Do we need to handle that case as well?

Reply:
In this case the context will be returned as is if it's ajax, which is ok because form objects returned in this context are not going to be re-rendered after post, so no need to include them.